testing/jaeger/client.go

```diff
@@ -73,10 +73,17 @@ func (j *Client) Traces(ctx context.Context, since time.Time) ([]Trace, error) {
 
 func AssertTag(t *testing.T, tags []Tag, k, v string) {
 	t.Helper()
+	if HasTag(tags, k, v) {
+		return
+	}
+	t.Errorf("key:%q with value %q was not found", k, v)
+}
+
+func HasTag(tags []Tag, k, v string) (bool, ) {
```

Comment:
`(bool, )` was this an artifact of refactoring? Or is this a Go idiom I've not seen before?

Reply:
refactor fuckup